html/index.html

```diff
@@ -179,8 +179,10 @@ <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
                                     href="https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/"
                                     class="link-dark">Nautobot Firewall
                                     Models</a><span class="badge text-bg-dark rounded-pill ms-1">App</span></li>
-                            <li class="list-group-item">Nautobot ChatOps<span
-                                    class="badge text-bg-dark rounded-pill ms-1">Coming soon</span></li>
+                            <li class="list-group-item"><a
+                                    href="https://docs.nautobot.com/projects/chatops/en/latest/"
+                                    class="link-dark">Nautobot ChatOps</a><span
+                                    class="badge text-bg-dark rounded-pill ms-1">App</span></li>
                             <!-- <li class="list-group-item"><a
```

Comment:
Would you please also remove the commented `li` below as it's no longer needed?

Reply:
Sure